app/src/release/java/org/mozilla/focus/utils/AdjustHelper.java

```diff
@@ -19,7 +19,12 @@
 import org.mozilla.focus.FocusApplication;
 import org.mozilla.focus.telemetry.TelemetryWrapper;
 
+import java.lang.ref.WeakReference;
+
 public class AdjustHelper {
+
+    private static WeakReference<FocusApplication> weakApplication;
```

Comment:
`Application` instance lifecycle is like static, it doesn't make sense to have it in a WeakReference